crates/cli/src/main.rs

```diff
@@ -375,6 +375,23 @@ impl Commands {
 
                 let pipeline = resolve_pipeline(pipeline)?;
                 let pipeline_file_name = pipeline.file_name().unwrap();
+
+                #[cfg(feature = "telemetry")]
+                let telem_join = segment_enabled().then(|| {
+                    let pipeline_name = pipeline_file_name.to_string_lossy().to_string();
```

Comment:
Lets make this a salted hash!
- 1. store a local salt like anon id
- 2. make a sha of the pipeline name with the salted hash simple as `hash(format!("{name}{salt}"))`
- 3. profit by not leaking names!

Reply:
Also use sha256